README.md

```diff
@@ -27,6 +27,7 @@ very secret.
 | run the site so that devices on your local network can access it  | `npm run watch -- --host 0.0.0.0`. Note that we use CORS to limit what hosts can access different APIs, so accessing with a `192.168.x.x` address may run into problems |
 | run all tests | `npm run test` |
 | run only unit tests | `npm run test:unit` |
+| run only unit tests for a subset of tests | `./node_modules/.bin/mocha path/to/my/test.unit.spec.jsx` use `--recursive` to test an entire directory (assumes current working dir of `/vets-website`) |
```

Comment:
I guess I wasn't super clear in my description. To use `--recursive` I think you have to use wildcards like:

```
./node_modules/.bin/mocha --recursive 'path/to/my/**/*.unit.spec.js?(x)'
```